### PR TITLE
KAFKA-8208: Fix broken link for out of order data

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

**Comment by bbejeck** [comment]
\cc @MichaelDrogalis

**Comment by MichaelDrogalis** [comment]
LGTM

File: 21/streams/core-concepts.html [hunk 1]
```diff
@@ -224,7 +224,7 @@ <h3><a id="streams_out_of_ordering" href="#streams_out_of_ordering">Out-of-Order
 
     <p>
         Besides the guarantee that each record will be processed exactly-once, another issue that many stream processing application will face is how to
-        handle <a href="tbd">out-of-order data</a> that may impact their business logic. In Kafka Streams, there are two causes that could potentially
+        handle <a href="https://www.confluent.io/wp-content/uploads/streams-tables-two-sides-same-coin.pdf">out-of-order data</a> that may impact their business logic. In Kafka Streams, there are two causes that could potentially
```
Comment (mjsax) [inline 21/streams/core-concepts.html]: We should not link to CP webpage from AK. Also, I am wondering if linking to the paper was the goal? If yes, we can also link to https://dl.acm.org/citation.cfm?id=3242155

**Comment by bbejeck** [comment]
@mjsax @MichaelDrogalis  updated the link

**Comment by MichaelDrogalis** [comment]
Good to me.

**Comment by bbejeck** [comment]
\cc @JimGalasyn

**Comment by JimGalasyn** [comment]
cc @londoncalling

**Review by londoncalling — APPROVED** [review]
@JimGalasyn LGTM!

**Comment by bbejeck** [comment]
Merged #197 into asf-site

**Pull request merged**